### ISSUE TITLE
Ability to set 'ignore RX', set contrast, set RGB without writing EEPROM

### ISSUE THREAD

**Issue by JamesB7** [sparkfun/OpenLCD#35]
In my application, I do not need the device to write to EEPROM -- really, for anything. I expect to be streaming 'set RGB' quickly to do fades between colors (for example, fade white to red when an error occurs, fade back afterward), and this will certainly wear the EEPROM out if it writes it.

It would be very helpful to have a pair of commands to disable/re-enable EEPROM writes when changing settings. (These would be sent during initialization and so should not themselves be saved.)

Thank you!

James

**Comment by JamesB7** [comment]
Would you accept a patch for this if I were to write one?
I'd do it by adding commands to the pipe command set -- there's plenty of room there, and I can do this simply.
(A command of this sort would also protect the device against bricking. If bits flip, and it writes a new I2C address to EEPROM, for example, no more screen. If I put the device in a 'EEPROM write disable' state *most* of the time, odds are, end users will not run into a problem.)

**Comment by nseidle** [comment]
Yes please, PRs are welcome. Our greatest priority is to maintain reverse compatibility so as long as it's new commands, we should be fine.
